packages/mobile/src/send/SendAmount.tsx

```diff
@@ -206,6 +215,21 @@ export class SendAmount extends React.Component<Props, State> {
     }
   }
 
+  isPaymentLimitReached = (now: number) => {
+    // we are only interested in the last 24 hours
+    const paymentsLast24Hours = this.props.recentPayments.filter(
+      (p: PaymentInfo) => timeDeltaInHours(now, p.timestamp) < 24
+    )
+
+    const initial = new BigNumber(this.state.amount)
+    const amount: BigNumber = paymentsLast24Hours.reduce(
+      (sum, p: PaymentInfo) => sum.plus(p.amount),
```

Comment:
Keep in mind that the amount is entered in a local currency, whereas the hard limit is in USD. If you need help figuring out the conversion utilities I think @jeanregisser can help :)

Reply:
@jeanregisser I am now using this.getDollarsAmount local method. Is that a good choice?